Documentation/platforms/arm64/zynq-mpsoc/boards/zcu111/index.rst

```diff
@@ -2,6 +2,8 @@
 Zynq UltraScale+ RFSoC ZCU111
 =============================
 
+.. tags:: chip:xczu28dr
```

Comment:
+ chip:fpga ? :-)

Reply:
I like this idea! I wonder if just `fpga` would suffice? Since that is not really a specific chip/chip family but more of a descriptor about the architecture (although not quite something like `arch:arm`, still different)? 

Any ideas maybe about a prefix I could use for this to differentiate from chip name but still convey a "class" of chip? I was thinking of doing the same thing for the virtual platforms, like `virt` or `emulator`. I can add this along with the other changes in the morning if you have any ideas!

Reply:
FPGA would be the general name, subclass could include vendor (i.e. `vendor:amd, vendor:xilinx`) and/or chip family (i.e. `chip:fpga, chip:zynq, chip:ultrascale+` not sure if `+` mark can be part of tag name) ? :-)